kernel/kselftest.py

```diff
@@ -81,7 +81,7 @@ def setUp(self):
         if self.run_type == 'upstream':
             location = self.params.get('location', default='https://github.c'
                                        'om/torvalds/linux/archive/master.zip')
-            tarball = self.fetch_asset("kselftest.zip", locations=location,
+            tarball = self.fetch_asset("kselftest.zip", locations=[location],
```

Comment:
earlier statement also seems me proper , so i don't think so this patch needed :) if you thinking to make this change as style issue can you please change commit message and  this patch can include other test case as well where the same case

Reply:
@PraveenPenguin `self.fetch_asset` requires a list as input for location. Existing code had an issue which eventually hits the following issue 
```
2019-06-26 05:39:28,308 stacktrace       L0044 ERROR| Reproduced traceback from: /usr/lib/python2.7/site-packages/avocado_framework-68.0-py2.7.egg/avocado/core/test.py:829
2019-06-26 05:39:28,317 stacktrace       L0047 ERROR| Traceback (most recent call last):
2019-06-26 05:39:28,317 stacktrace       L0047 ERROR|   File "/home/OpTest/avocado-fvt-wrapper/tests/avocado-misc-tests/kernel/kselftest.py", line 89, in setUp
2019-06-26 05:39:28,317 stacktrace       L0047 ERROR|     expire='1d')
2019-06-26 05:39:28,317 stacktrace       L0047 ERROR|   File "/usr/lib/python2.7/site-packages/avocado_framework-68.0-py2.7.egg/avocado/core/test.py", line 1073, in fetch_asset
2019-06-26 05:39:28,317 stacktrace       L0047 ERROR|     self.cache_dirs, expire).fetch()
2019-06-26 05:39:28,317 stacktrace       L0047 ERROR|   File "/usr/lib/python2.7/site-packages/avocado_framework-68.0-py2.7.egg/avocado/utils/asset.py", line 184, in fetch
2019-06-26 05:39:28,317 stacktrace       L0047 ERROR|     ": %s" % urlobj.scheme)
2019-06-26 05:39:28,317 stacktrace       L0047 ERROR| UnsupportedProtocolError: Unsupported protocol:
```
as it tries to fetch from a string (url in this case). So I would like to keep the commit as is.

Reply:
ah really weird let me check if i can do any thing with asset fetch lib , can you please include other test as well

Reply:
ah i got my mistake , sorry for unnecessary noise :(